distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -220,7 +220,7 @@ public void testCswRegistryStoreUpdate() throws Exception {
                         FederationAdminService.class);
                 federationAdminServiceImpl.updateRegistryEntry(Library.getRegistryNode(regID,
                         "New Node Name",
-                        "2015-02-26T17:16:34.996Z"), destinations);
+                        "2016-02-26T17:16:34.996Z"), destinations);
```

Comment:
For the future, when it comes to dates in tests, it is always best to generate dates yourself, lest you run into issues down the road. See the security itests for examples.